code/game/objects/items/storage/belt.dm

```diff
@@ -1296,12 +1296,12 @@
 #undef MAXIMUM_MAGAZINE_COUNT
 
 /obj/item/storage/belt/gun/m44
-	name = "\improper M276 pattern M44 holster rig"
+	name = "\improper M276 pattern general revoler holster rig"
 	desc = "The M276 is the standard load-bearing equipment of the USCM. It consists of a modular belt with various clips. This version is for the M44 magnum revolver, along with six small pouches for speedloaders. It smells faintly of hay."
```

Comment:
should maybe change this to include all revolvers instead of just mentioning the m44